constructors/library/constructors/constructors_version.js

```diff
@@ -0,0 +1,10 @@
+// -----------------------------------------------------------------------
+// --
+// -- Copyright (C) 2021 CARTO
+// --
+// -----------------------------------------------------------------------
+
+/* exported constructorsVersion */
+function constructorsVersion() {
+    return '1.0.0';
```

Comment:
Since you're adding a JS lib with a version function too I think you should use it here. (this is the kind of thing that @Jesus89 code generation tool will save us from)

Reply:
Yes, this is autogenerated code from the setool

Reply:
oops, then @Jesus89 I would have expected it to use the JS lib if it's generated